bundle/manifests/mcg-osd-deployer-df-dedicated-admin-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    managed.openshift.io/aggregate-to-dedicated-admins: cluster
+    olm.owner.kind: OperatorGroup
+  name: mcg-osd-deployer-df-dedicated-admin-role
+rules:
+- apiGroups:
+  - noobaa.io
```

Comment:
we should add group for OB/OBC as well IMO, wdyt ?
it do not belong to noobaa.io

Reply:
@SanjalKatiyar  user is not creating OB/OBC directly from the console, in that case, I think we don't need that

Reply:
he will if he go to `Storage > ObjectBucketClaims > Create OBC` and create from here...